ui/components/ui/logo/logo.stories.js

```diff
@@ -45,7 +45,7 @@ const LogoItem = ({ Component }) => {
       backgroundColor={BackgroundColor.backgroundDefault}
     >
       <Box marginBottom={2}>{Component}</Box>
-      <Text>{`${Component.type.__docgenInfo.displayName}`}</Text>
+      <Text>{`${Component.type.name}`}</Text>
     </Card>
```

Comment:
Was this just deprecated? Don't see any visual changes

<img width="1506" alt="Screenshot 2024-02-16 at 8 49 22 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/127cda41-561c-4873-bf19-56330a770325">

<img width="1507" alt="Screenshot 2024-02-16 at 8 49 02 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/729eb2c6-1549-4ac0-a8e8-95ddb196d57c">

Reply:
I'm not completely sure, but once @davidmurdoch turned on useSWC, this stopped working without changing `Component.type.__docgenInfo.displayName` to `Component.type.name`.